nav2_mppi_controller/include/nav2_mppi_controller/critic_data.hpp

```diff
@@ -44,10 +39,10 @@ namespace mppi
 struct CriticData
 {
   const models::State & state;
-  const models::Trajectories & trajectories;
+  models::Trajectories & trajectories;
```

Comment:
Why removed const-ness?

Reply:
In critic files(ex path align critic), I was passing non-const in the template argument of the Eigen::Map, hence it was expecting non-const trajectories. I have corrected it now.